### PR TITLE
Update /download/cloud copy

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update /download/cloud copy
- Created an include from a strip on /public-cloud and used it on both pages
- Updated logos

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/cloud
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to and resolve comments in the [copy doc](https://docs.google.com/document/d/1kp7QQXst1EbCjZjzWZP3YZwfmG0EIXLfQoRUVaMEZng/edit?ts=5f6b774e#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3213

### CODE REVIEW

**Review by anthonydillon — CHANGES_REQUESTED** [review]
## QA
- The hero image looks squished
image

File: templates/download/cloud/index.html [hunk 1]
```diff
@@ -36,73 +38,75 @@ <h2>Use Ubuntu in the cloud</h2>
...
+        <li class="p-list__item u-no-margin--top"><a href="https://help.ubuntu.com/community/EC2StartersGuide" class="p-link--external">Ubuntu's guide to using EC2</a></li>
+        <li class="p-list__item"><a href="http://cloud-images.ubuntu.com/locator/ec2/" class="p-link--external">Amazon EC2 AMI Locator</a></li>
+        <li class="p-list__item"><a href="https://ubuntu.com/aws/pro">Launch Ubuntu Pro for AWS&nbsp;&rsaquo;</a></li>
       </ul>
     </div>
     <div class="p-card col-4">
       <div class="p-card__content">
         <h4>
-          <a class="u-align--center u-vertically-center" style="min-height:10rem;display: flex;" href="http://cloud-images.ubuntu.com/locator/ec2/">
+          <a class="u-align--center u-vertically-center" style="min-height:10rem;display: flex;" href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/Canonical.UbuntuServer">
```
Comment (anthonydillon) [inline templates/download/cloud/index.html:66]: The copy doc asks for this to link to ubuntu.com/azure

File: templates/shared/_ubuntu_pro_on_public_cloud.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<section class="p-strip--light is-deep is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-8 p-card">
+      <h2>Ubuntu Pro on public cloud</h2>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/shared/_ubuntu_pro_on_public_cloud.html:4]: actually, that is out of date, so I removed it and updated the copy docs

**Comment by pmahnke** [comment]
@anthonydillon ready for a relook

**Review by anthonydillon — APPROVED** [review]
LGTM

**Pull request merged**